mux.js

```diff
@@ -71,6 +72,11 @@ function Multiplexer (key, opts) {
       case MANIFEST:
         var rm = JSON.parse(message.toString('utf8'))
         self._remoteHas = rm.keys
+
+        // TODO: Throttle this emit
+        // not sure how to handle misbehaving clients.
+        // Should we allow buffering of remote feed-announcements and combine them into a single manifest?
+        // or should we just ignore any manifests that were sent to fast?
```

Comment:
Suggestion: turn all of this throttling & peer abuse Qs into an issue to tackle later?

Reply:
Yeah you're entirely correct, my ambitions went ahead of me. Throttling removed for now.